test/extended/networking.sh

```diff
@@ -24,7 +24,6 @@ NETWORKING_E2E_MINIMAL="${NETWORKING_E2E_MINIMAL:-}"
 
 DEFAULT_SKIP_LIST=(
   # TODO(marun) This should work with docker >= 1.10
-  "openshift router"
   "\[Feature:Federation\]"
```

Comment:
The federation e2e suite requires extensive setup, so they should be skipped. I'm not sure why they would be selected with a Networking focus anyway.  

The router tests should work, though I haven't checked recently.  I'm not sure there's value in running those tests multi-node though.  They are already run single-node as part of the conformance suite.